winrm/certgen.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2016 hoenirvili
```

Comment:
Please remove this copyright attribution. This project doesn't have any copyright attribution at the file-scope level, as this is fully recorded in the Git log.